charts/s3gw/templates/ingress-traefik.yaml

```diff
@@ -12,12 +12,12 @@ metadata:
     traefik.ingress.kubernetes.io/router.tls: "true"
     traefik.ingress.kubernetes.io/router.middlewares:
       '{{ .Release.Namespace }}-{{ include "s3gw.CORSMiddlewareName" . }}@kubernetescrd'
-    cert-manager.io/cluster-issuer: {{ default .Values.tlsIssuer .Values.customTlsIssuer | quote }}
+    cert-manager.io/cluster-issuer: {{ .Release.Name }}-{{ .Release.Namespace }}-{{ .Values.tlsIssuer }}
 spec:
   tls:
     - hosts:
         - '{{ include "s3gw.serviceName" . }}.{{ .Values.publicDomain }}'
-        - '*.{{ include "s3gw.serviceName" . }}.{{ .Values.publicDomain }}'
+#        - '*.{{ include "s3gw.serviceName" . }}.{{ .Values.publicDomain }}'
```

Comment:
Maybe @m-ildefons can comment on this, but I think this will not work. We need wildcard certificates for vhost-based bucket access.

Reply:
Thanks @jecluis , we are discussing this wildcard point inside our `discuss-s3gw` Slack channel